scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -423,6 +427,43 @@ <h4 class="modal-title">Edit synopsis</h4>
   </div>
 {% endmacro %}
 
+{% macro clinvar_panel() %}
+  <div class="panel panel-default">
+    <div class="panel-heading">Variants submitted to clinvar</div>
```

Comment:
"Clinvar submissions" perhaps - could be listing unsubmitted ones as well?

Reply:
Yes, of course. The only way I have to know if it was submitted is that the variant has been published and the clinvar ID is saved in scout database. But we want to keep it broad here. Then I'll change it also in the cases page where under "Clinvar" the label says "submitted". I'll write instead "Clinvar submissions" and the label "created"

Reply:
Right, thanks! Otw agree with Måns we should have a few tests - perhaps at least for the mongo adapter part?